### ISSUE TITLE
Metalhead guass got its beat back

### ISSUE THREAD

**Issue by ArchShaman** [ZeroK-RTS/Zero-K#3349]
http://zero-k.info/Battles/Detail/645219
@Anarchid

**Comment by Anarchid** [comment]
Cant watch the game for a while; what are the conditions, can you do this on purpose?

**Comment by ArchShaman** [comment]
Gauss was placed on the eastern hill near the ramp. It was elevated and tried firing at things below it (I think).

**Comment by sprunk** [comment]
Duplicate of #3550

**Comment by Anarchid** [comment]
I was testing lasercannon gauss for #2414 and got a low-frequency version of this. Since the shifting geometry is now completely ruled out by https://github.com/ZeroK-RTS/Zero-K/commit/e32ea20670fbad0c1063f3c55d61b55d91948bd7,  this must be some kind of logic error in the script. 

The few cases where it triggered were when i told the turret to aim in a way that required it to change both pitch and heading while playing the pack-down animation.

It also consistently triggers something *similar* but not identical to metalheading when it does that, in a non-pathological way. I think each call to aiming triggers another unpack process, which doesn't fundamentaly work with animations that have intermediate steps = cannot be easily interpolated. 

Perhaps a solution is to store the turret's animation status (packing, unpacking) somewhere and don't trigger unpack animations when already unpacking.

Using this issue rather than #3550 because 3550 has the shifting geometry as its core. *That* has been dealt with; *this*, not.

**Comment by SteelBlueZK** [comment]
Similar to https://github.com/ZeroK-RTS/Zero-K/issues/3550 recommend closing.

**Comment by sprunk** [comment]
#3550 is already closed though. Apparently jitter still exists sometimes so a ticket should be kept open. Let's close after #4329 gets merged.

**Comment by sprunk** [comment]
62c7a2494e2242140f7cd8cd858de4e4b0690213
